tests/unix/test_lwt_unix.cppo.ml

```diff
@@ -677,11 +677,28 @@ let bind_tests = [
         (fun () -> Lwt_unix.close socket));
 ]
 
+let dir_tests = [
+    test "getcwd"
+    (fun () ->
+      Lwt_unix.getcwd () >>= fun (_:string) ->
+      Lwt.return_true
```

Comment:
One thing that we can do for the basic test, is compare the result of `Lwt_unix.getcwd` to the result of `Unix.getcwd` from the stdlib.